### ISSUE TITLE
Difference with Earth2-mip

### ISSUE THREAD

**Issue by ShihengDuan** [NVIDIA/earth2studio#148]
Hi,

Thanks for putting together all these fantastic AI weather models. 

I'm wondering what's the difference between Earth2-mip and Earth2-studio. Will the Earth2-mip still be maintained alongside with Earth2-studio?

**Comment by dallasfoster** [comment]
TLDR; Earth2Studio is bring-your-own model, bring-your-own data, bring-your-own workflow. Earth2MIP is primarily bring-your-own model.

To elaborate

The philosophy behind Earth2Studio is to provide building blocks for users to build their own workflows. In this package we focus on the following building blocks: data sources, model wrappers, IO handling, common utilities. Earth2MIP is primarily targeted at providing specific workflows, usually with specific metrics or plotting scripts associated with those workflows.

Earth2Studio is actively maintained and has ongoing software engineering support.
